wiki/Legal/Privacy/ru.md

```diff
@@ -0,0 +1,120 @@
+---
+legal: true
+---
+
+# Политика конфиденциальности osu!
+
+Последнее обновление: 25 мая 2018. [Посмотреть историю](https://github.com/ppy/osu-wiki/commits/master/wiki/Legal/Privacy/en.md)
```

Comment:
this should probably say something like "Original version last updated:", since the date and link are referring to English